lib/spinach/runner.rb

```diff
@@ -70,14 +70,21 @@ def run
       successful = true
 
       filenames.map do |filename|
-        filename.split(':')
-      end.each do |filename, line|
+        file = filename.split(':')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.